arm-apimanagement/2015-09-15/swagger/service.json

```diff
@@ -0,0 +1,1288 @@
+        "tags": [
+          "ApiManagement"
+        ],
+        "operationId": "ApiManagement_Get",
+        "description": "Get ApiManagement service (resource description).",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIManagementServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/APIManagementService"
+            }
+          },
+          "404": {
+            "description": "APIManagement service Not Found",
+            "schema": {
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ApiManagement"
+        ],
+        "operationId": "ApiManagement_CreateOrUpdate",
+        "description": "Create APIManagement service, or update an existing service",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIManagementServiceNameParameter"
+          },
+          {
+            "name": "service",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/APIManagementService"
+            },
+            "description": "Parameters supplied to the CreateOrUpdate ApiManagement service operation.",
+            "x-nullable": true
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/APIManagementService"
+            }
+          },
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/APIManagementService"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "ApiManagement"
+        ],
+        "operationId": "ApiManagement_Delete",
+        "description": "Deletes APIManagement service. This operation takes a while to complete.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIManagementServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
+          },
+          "204": {
+            "description": ""
```

Comment:
As mentioned in the description, it takes a while. So this should be marked as long running